content/docs/04-clusters/03-edge/04-edgeforge-workflow/03.5-palette-canvos.md

```diff
@@ -0,0 +1,658 @@
+<br />
+
+  ```shell
+  docker images
+  ```
+
+  ```bash coloredLines=3-4
+  # Output
+  REPOSITORY        TAG                                IMAGE ID       CREATED         SIZE
+  ttl.sh/ubuntu     k3s-1.25.2-v3.4.3-palette-learn    b3c4956ccc0a   6 minutes ago   2.49GB
+  ttl.sh/ubuntu     k3s-1.24.6-v3.4.3-palette-learn    fe1486da25df   6 minutes ago   2.49GB
+  earthly/earthly   v0.7.4                             d771cc8edc38   2 weeks ago     333MB
+  ```
+  <br />
+
+
+12. To use the provider OS images in your cluster profile, push them to the image registry mentioned in the **.arg** file. The current example and default behavior uses the [ttl.sh](https://ttl.sh/) image registry. This image registry is free to use and does not require a sign-up. Images pushed to *ttl.sh* are ephemeral and will expire after the 24 hrs time limit.  Use the following commands to push the provider OS image to the *ttl.sh* image registry.  
+  <br />
+
+  ```bash
+  docker push ttl.sh/ubuntu:k3s-1.25.2-v3.4.3-palette-learn
+  docker push ttl.sh/ubuntu:k3s-1.24.6-v3.4.3-palette-learn
+  ```
+  <br />
+
+  <WarningBox>
+
+  As a reminder, [ttl.sh](https://ttl.sh/) is a short-lived image registry. If you do not use these provider OS images in your cluster profile within 24 hours of pushing to *ttl.sh*, they will expire and must be re-pushed. Refer to the Advanced workflow in the current guide to learn how to use another registry, such as Docker Hub, and tag the docker images accordingly.
+
+  </WarningBox>
+  <br />
+
+
+13. After pushing the provider images to the image registry, add the following custom content within the `options` attribute of your cluster profile's **OS layer** so that the cluster profile can pull the provider OS image from the image registry. 
+
+  If you used customized arguments and Dockerfile to build the Edge artifacts, change these attributes' values, as applicable to you, before using them in your cluster profile.
+  <br />
+
+  ```yaml
+    system.uri: "{{ .spectro.pack.edge-native-byoi.options.system.registry }}/{{ .spectro.pack.edge-native-byoi.options.system.repo }}:{{ .spectro.pack.edge-native-byoi.options.system.k8sDistribution }}-{{ .spectro.system.kubernetes.version }}-{{ .spectro.pack.edge-native-byoi.options.system.peVersion }}-{{ .spectro.pack.edge-native-byoi.options.system.customTag }}"
+    system.registry: ttl.sh
+    system.repo: ubuntu
+    system.k8sDistribution: k3s
+    system.osName: ubuntu
+    system.peVersion: v3.4.3
+    system.customTag: demo
+    system.osVersion: 22
+  ```
+  The screenshot below shows custom content added to a cluster profile's OS layer.
+
+  ![Screenshot of a sample cluster profile's OS layer ](/tutorials/palette-canvos/clusters_edge_palette-canvos_edit_profile.png)
+  <br />
+
+  After updating the OS layer, you must choose a Kubernetes version compatible with the provider OS image in your cluster profile. 
```

Comment:
We can't stop here, have the user select a Kubernetes version and emphasize on using one of the versions created by the build process. And lastly, guide them in choosing a CNI but explain that they must choose a CNI that best fits their need.